webpack.config.js

```diff
@@ -46,11 +51,4 @@ const config = {
   }
 };
 
-module.exports = () => {
-  if (isProduction) {
```

Comment:
can be simplified in one line: `mode: isProduction ? 'production' : 'development',` directly in config object